keanu-project/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,10 @@
+# Root logger option
+log4j.rootLogger=INFO, stdout
+log4j.logger.io.improbable.keanu=INFO
+log4j.logger.org.nd4j=DEBUG
```

Comment:
shall we whack all of these up to WARN as the default?

Reply:
Yeah, I think that would make sense, but wasn't sure what defaults others prefer.

Reply:
We do have a bunch of logs that are on INFO level, these would not be displayed if we change it to WARN. Do you think that'd be fine?

Reply:
I doubt anyone would miss them. Worth getting 1 or 2 more opinions though.

Reply:
I think WARN by default is fine. If anyone wants to particularly probe into whats happening they can override it.